api/openapi-spec/swagger.json

```diff
@@ -45699,7 +45699,44 @@
     }
    },
    "io.k8s.apimachinery.pkg.apis.meta.v1.Patch": {
-    "description": "Patch is provided to give a concrete name and type to the Kubernetes PATCH request body."
+    "description": "Patch is provided to give a concrete name and type to the Kubernetes PATCH request body.",
+    "type": "array",
+    "items": {
+       "$ref": "#/definitions/io.k8s.apimachinery.pkg.apis.meta.v1.PatchDocument"
+      }
+   },
+    "io.k8s.apimachinery.pkg.apis.meta.v1.PatchDocument": {
+    "description": "A JSONPatch document as defined by RFC 6902.",
```

Comment:
I'm not very familiar with our swagger doc, but wouldn't this be the spot to describe the two other patch types?